traffic_ops/traffic_ops_golang/urisigning/urisigning.go

```diff
@@ -62,7 +63,7 @@ func GetURIsignkeysHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	if len(ro) == 0 {
-		ro, err = json.Marshal(tc.URISignerKeyset{})
+		ro, err = json.Marshal(jwk.NewSet())
```

Comment:
For backwards compatibility, when there are no  `urisigning.GetURIsignkeysHandler()` should return
```json
{
    "renewal_kid": null,
    "keys": null
}
```

when there are no URI signing keys (see apache/trafficcontrol#5884).

Reply:
I believe this is just a problem in this particular path, so I opted to just stick a dummy struct so that we don't have to play games https://github.com/apache/trafficcontrol/pull/6380/commits/b947d63baa1adac7124f12b387ae6a54c531613c